src/node_buffer.cc

```diff
@@ -1072,6 +1072,42 @@ void IndexOfNumber(const FunctionCallbackInfo<Value>& args) {
                                 : -1);
 }
 
+void SwapShort(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  THROW_AND_RETURN_UNLESS_BUFFER(env, args.This());
+  SPREAD_ARG(args.This(), ts_obj);
+
+  if (ts_obj_length % 2 != 0)
```

Comment:
there's a nagging part of me that wants to have `ts_obj_length % sizeof(uint16_t)`, but have no good reason to justify it.